scripts/runDocker.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+NAME=`docker load -i builds/*docker* | awk '{ print $3}'`
```

Comment:
Need to investigate how this can be done inline.

Reply:
This has been fixed and the old script removed.